packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -1294,6 +1306,51 @@ export default class SalesforceCRMService implements CRM {
     return null;
   }
 
+  private async getPicklistFieldValue({
+    fieldConfigValue,
+    salesforceField,
+    calEventResponses,
+    bookingUid,
+    contactId,
+  }: {
+    fieldConfigValue: string;
+    salesforceField: Field;
+    calEventResponses?: CalEventResponses | null;
+    bookingUid?: string | null;
+    contactId: string;
+  }) {
+    const log = logger.getSubLogger({ prefix: [`[getPicklistFieldValue] ${contactId}`] });
+
+    const picklistValues = salesforceField.picklistValues;
+
+    if (!picklistValues || !picklistValues.length) {
+      log.warn(`No picklist values found for field ${salesforceField.name}`);
+      return null;
+    }
+
+    // Get the text value from the field
+    const fieldTextValue = await this.getTextFieldValue({
```

Comment:
We use this to handle if we want to get the value from a booking question or a routing form